### PR TITLE
[12.0][ADD] connector_acp and connector_acp_ftp

### PR DESCRIPTION
**Pull request by nikitavaghela**
Improved connector_acp module as per #57  and added connector_acp_ftp module as per #58 .

### CODE REVIEW

**Review by nelsonramirezs — APPROVED** [review]

**Comment by nikitavaghela** [comment]
@dreispt Can you please review this PR?

File: connector_acp/models/etd_mixin.py [hunk 1]
```diff
@@ -52,74 +54,93 @@ def get_etd_document(self):
...
+                self.env['ir.attachment'].create({
+                    'name': res_file.filename,
+                    'type': 'binary',
+                    'datas':
+                        base64.b64encode(res_file.filecontent.encode("utf-8")),
+                    'datas_fname': res_file.filename,
+                    'res_model': rec._name,
+                    'res_id': rec.id})
+            file_list.append({'name': filename, 'content': res_content})
+        return ", ".join(repr(e) for e in file_list)
```
Comment (dreispt) [inline connector_acp/models/etd_mixin.py:124]: I think this should be:
```suggestion
        return file_list
```

**Comment by dreispt** [comment]
@nikitavaghela I'm not comfortably merging this, even if in Beta - it may break existing features.
You're welcome to incorporate it in the next PR you are working on. Thanks!

**Comment by dreispt** [comment]
Superseded by #73 and #65.
Thank you @nikitavaghela

**Pull request closed without merging**